Added global status bar (so the clock, battery etc. is not hidden beh…

anand-ts opened the pull request:
- Added global status bar in App.js (to unhide battery status, clock, wifi indicator etc.)
- Changed background image
- Added Header for AppointmentsPage.js
- Other headers coming soon

anand-ts commented:
- Updated styles for BusinessInfoPage.js
- Added header for the page as well

DavidHe137 commented on beauty-consulting-app/app/Pages/AppointmentsPage.js:
The header on this page does not have great spacing on my phone (iPhone SE), which is smaller than most phones. To make it easier for our styles to work on all screen sizes, would it be easier to make the header completely fill up to the top instead of being a floating island? I think it might also look better.

Here's what it looks like on my phone:
![IMG_4392](https://github.com/user-attachments/assets/7aa40b7a-4b5f-44f3-a68d-db1ba71c8f3a)

DavidHe137 reviewed (changes requested):
Nice changes! The BusinessInfo Page looks much better! I definitely have more notes on appearance, but I think this the changes in here should be sufficient for a PR. 

My main note is about the header, and let me know if you disagree: I think the header should completely fill the top instead of being a floating island. An example header might look like it does in Discord, albeit a different color:
![IMG_72C828BF2265-1](https://github.com/user-attachments/assets/0340330a-0df2-44fe-a84d-0c02733d3d93)

anand-ts commented on beauty-consulting-app/app/Pages/AppointmentsPage.js:
```diff
@@ -1,3 +1,4 @@
+// AppointmentsPage.js
```
Yeah that's my bad, I will remove those remaining commented code.

anand-ts commented:
- Added focus aware status bar, so it's dark on white backgrounds and vice-versa
- Added icons on the bottom tab navigator to remove triangles
- Changed KeyboardMove component padding to "0" to remove white space in sign in page
